### PR TITLE
Implemented Error Messages for SignIn and SignUp

### PR DESCRIPTION
**Pull request by tusharraval102**
This PR is related to creating new modals to display error for the following scenarios: 

- Password too short
- Wrong password
- wrong email
- email already exists
- email is in use for the other systems

### CODE REVIEW

File: Admin-app/src/components/SignIn/index.js [hunk 1]
```diff
@@ -183,7 +180,22 @@ export default function SignIn() {
                 Error
               </Typography>
               <Typography id="modal-modal-description" sx={{ mt: 2 }}>
-                You are currently in the Administration Application, please sign in using this email on the Client App.  
+                This email is not registered with the Administration application.
+              </Typography>
+            </Box>
+          </Modal>
+          <Modal
```
Comment (SamanthaGuillemette) [inline Admin-app/src/components/SignIn/index.js:187]: @tusharraval102  can you create a component for this  please?

File: Admin-app/src/components/SignUp/index.js [hunk 2]
```diff
@@ -314,6 +315,21 @@ export default function SignUp(props) {
               </Typography>
             </Box>
             </Modal>
+            <Modal
```
Comment (SamanthaGuillemette) [inline Admin-app/src/components/SignUp/index.js:318]: @tusharraval102 can you create component for here too, or same component that will be created for admin signin!

File: Client-app/src/components/SignIn/index.js [hunk 3]
```diff
@@ -152,10 +152,25 @@ export default function SignIn() {
                 Error
               </Typography>
               <Typography id="modal-modal-description" sx={{ mt: 2 }}>
-                You are currently in the Client Application, please sign in using this email on the Administration App.                
+                This email is not registered with the Administration application.
               </Typography>
             </Box>
             </Modal>
+            <Modal
```
Comment (SamanthaGuillemette) [inline Client-app/src/components/SignIn/index.js:159]: same here @tusharraval102, have a component for the modal please!

File: Client-app/src/components/SignUp/index.js [hunk 4]
```diff
@@ -289,6 +292,21 @@ export default function SignUp(props) {
               </Typography>
             </Box>
             </Modal>
+            <Modal
```
Comment (SamanthaGuillemette) [inline Client-app/src/components/SignUp/index.js:295]: here you can reuse the component you create in client signin @tusharraval102

**Review by SamanthaGuillemette — CHANGES_REQUESTED** [review]
Nice code! All looks good! Would it be possible for you @tusharraval102 to create separate components for the error modals themselves? That way it would be clearer and more modalized ! Similar to what Quang did for the profile edit modal!

File: Admin-app/src/components/SignIn/index.js (hunk 1 above)
Comment (tusharraval102) [reply Admin-app/src/components/SignIn/index.js:187]: Yup, ill do that now

**Pull request merged**